core/planner.h

```diff
@@ -55,7 +55,6 @@ extern const char *get_planner_disclaimer();
 extern void free_dps(struct diveplan *diveplan);
 extern struct dive *planned_dive;
 extern char *cache_data;
-extern char *disclaimer;
```

Comment:
The build failure on Windows seems to imply that this is still used somewhere in `mainwindow.cpp`

Reply:
Should be fixed.